pkg/kubelet/preemption/preemption.go

```diff
@@ -85,16 +85,16 @@ func (c *CriticalPodAdmissionHandler) HandleAdmissionFailure(pod *v1.Pod, failur
 		// Return only reasons that are not resource related, since critical pods cannot fail admission for resource reasons.
 		return false, nonResourceReasons, nil
 	}
-	err := c.evictPodsToFreeRequests(admissionRequirementList(resourceReasons))
+	err := c.evictPodsToFreeRequests(pod, admissionRequirementList(resourceReasons))
 	// if no error is returned, preemption succeeded and the pod is safe to admit.
 	return err == nil, nil, err
 }
 
 // freeRequests takes a list of insufficient resources, and attempts to free them by evicting pods
 // based on requests.  For example, if the only insufficient resource is 200Mb of memory, this function could
 // evict a pod with request=250Mb.
-func (c *CriticalPodAdmissionHandler) evictPodsToFreeRequests(insufficientResources admissionRequirementList) error {
-	podsToPreempt, err := getPodsToPreempt(c.getPodsFunc(), insufficientResources)
+func (c *CriticalPodAdmissionHandler) evictPodsToFreeRequests(pod *v1.Pod, insufficientResources admissionRequirementList) error {
```

Comment:
nit: lets name this and others `admitPod` or something other than pod

Reply:
+1

Reply:
ok, do you mean rename arg `pod` to `admitPod`?